services/cadip/config/cadip_session_ODataToSTAC_template.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Do we need to add this file to rs-demo in https://github.com/RS-PYTHON/rs-demo/tree/main/local-mode/main/config ?
Maybe you'll also need to add it here: https://github.com/RS-PYTHON/rs-demo/blob/main/local-mode/main/scripts/rspy-pre-actions.sh#L17
Can you do it and test the demos again ?

Reply:
No needed, since this is part of rs-server-cadip service.